canonical-websites/snapcraft.io
# Snap listing

# Done

Fixes https://github.com/canonical-websites/snapcraft.io/issues/761

- Updated markup, moved things around
- Used `p-strips` for spacing between sections
- Removed `u-margin*` and `u-padding*` classes
- Added some `u-margin*` and `u-padding*` classes

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account/snaps/SNAP_NAME/listing
- Use the form and ensure everything works and looks as expected.
